### PR TITLE
Milestone 4: Readme instructions

### PR DESCRIPTION
**Pull request by wickathou**
Milestone 4 submissions: Readme file updates with instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+<!-- ABOUT THE PROJECT -->
+# About The Project
+## Tic Tac Toe in Ruby
+This is a Tic Tac Toe game, developed following the OOP paradigm and built in Ruby.
+## How to play
+* Read all instructions the game prompts carefully
+* Player 1 chooses the symbol they want to play with, player 2 will get assigned the remaining symbol
+* A prompt appears asking for the position that player 1 wants to move to
+* Once player 1 selects, player 2 will be prompted to select a position
+* The game alternates between both players until the board is full or until someone wins
```
Comment (mosesogwo) [inline README.md]: What are the conditions for winning?
Consider using images to show winning combinations.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,53 @@
...
+## Features:
+### Built on Ruby
+- You can check the game using Ruby
+### Interactive prompt system
+**Microverse @microverse**
```
Comment (mosesogwo) [inline README.md]: Please, I don't understand this section completely.
Could you leave a comment to explain this to me? Thanks

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Good work on your readme @wickathou, @aracelicaes :clap:**
- You forgot to add a description to your PR... Please do it.
- See my comments on your readme file.

_**Kindly make the requested changes and submit another code review request.**_

**Any questions? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### Status: Changes Requested.

**Review by Forison — APPROVED** [review]
# Status:: Approved✔️
## Good job on addressing the issues raised earlier team😎 .
## Go ahead and merge and move to your next task🚀.
#### Happy coding⌨️
Addo Forison

**Comment by aracelicaes** [comment]
Project approved.

**Pull request merged**